lib/y2_module_consoletest.pm

```diff
@@ -36,7 +36,7 @@ sub post_run_hook {
 }
 
 sub test_flags {
-    return {fatal => 0};
+    return get_var('PUBLIC_CLOUD') ? {no_rollback => 1} : {};
```

Comment:
Hey! In your VR I can see that the only module running is `yast2_lan_device_settings`. So I would put that change there, whereas it doesn't really harm to keep it here.

Reply:
@rwx788 thanks for pointing this, I would say that it is expected to run more `yast2` test modules on public cloud in QAM. Than can we keep this over here?

Reply:
@rwx788 is this correct?
```suggestion
    return get_var('PUBLIC_CLOUD') ? {no_rollback => 1, fatal => 0} : {fatal => 0};
```

Reply:
good catch, from my perspective it is.